### DISCUSSION TITLE
Q: 想要浏览量+文章反应，独立于评论系统吗？ || Q: Do you want views + article reactions, independent of the comment system?

### DISCUSSION THREAD

**Discussion by DeityIng** [walinejs/waline#2027] [General]
如果想在每个页面显示 浏览量+给浏览者点赞，某些页面不需要评论部分，目前支持吗？

---
If you want to display pageviews + likes for viewers on each page, some pages do not need a comment section, is it currently supported?

**Comment by lizheming** [comment]
pageview 是可以的，具体见文档 https://waline.js.org/guide/features/pageview.html#%E5%8D%95%E7%8B%AC%E4%BD%BF%E7%94%A8

浏览者点赞我理解你说的应该是评论头部的 Reaction 部分，这个暂时不支持，目前可以通过 CSS 隐藏，具体见 discussion #2026 

---
pageview is available, see the document https://waline.js.org/guide/features/pageview.html#%E5%8D%95%E7%8B%AC%E4%BD%BF%E7%94%A8

I understand that you are talking about the Reaction part of the comment head. This is not supported for the time being. It can be hidden through CSS. For details, see discussion #2026

**Reply by DeityIng** [reply]
谢谢，我试试

---
Thanks, I'll try

**Reply by DeityIng** [reply]
不过，说起来，静态页面下面有了评论，评论模块都有评论点赞了，在评论模块的最前面加上类似微信里面 “点赞“ “转发“等社交按钮，把这个一起做了，这样大家只用waline就搞定常用的动态需求，多好。。期待。。😏

---
However, speaking of it, there are comments under the static page, and the comment module has comments and likes. Add social buttons such as "like" and "forward" in WeChat to the front of the comment module, and make this together, so that everyone can only use waline to handle common dynamic needs. How good it is. . expect. . 😏
